weed/topology/data_node.go

```diff
@@ -190,3 +192,19 @@ func (dn *DataNode) ToDataNodeInfo() *master_pb.DataNodeInfo {
 	}
 	return m
 }
+
+// GetVolumeIds returns the human readable volume ids limited to count of max 100.
+func (dn *DataNode) GetVolumeIds() string {
+	volumesLen := len(dn.volumes)
+	if volumesLen > 100 {
+		return "..."
```

Comment:
how about showing the first 99 volume ids and one `...`?

Reply:
good idea